spec/context_formatter_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'spec_helper'
+
+describe Poesie::ContextFormatter do
+  before do
+    Poesie::Log::quiet = true
+  end
+
+  let(:terms) do
+    JSON.parse(fixture('terms.json'))
+  end
+
+  module CurrentOS
+    UNKNOW = 0
+    ANDROID = 1
+    IOS = 2
+  end
```

Comment:
You shouldn't had to redeclare this module here, it should have been declared in a file that is imported both in the lib and in the specs.

Now that we'll migrate to 2 constants for `XXX_FILTER_REGEX`, the same apply, you should declare those constants in `lib/poesie.rb` (and not in `bin/poesie`) so that everybody using the poesie "library", either from the binary wrapper or from the spec (or as a ruby gem in a ruby script if people want to do that in their own scripts too), the constants are know to everybody.